Calculator/Program.cs

```diff
@@ -9,14 +9,13 @@ static void Main(string[] args)
         {
             var currentUser = User.GetUsername();
             User.GreetUser(currentUser);
-
-            var calculationChoice = Operations.GetCalculationType(); 
+ 
             var firstNumber = Operations.GetFirstNumber(); 
-            var secondNumber = Operations.GetSecondNumber(); 
+            var secondNumber = Operations.GetSecondNumber();
 
-            var sumOfNumbers = Operations.CalculateSumOfNumbers(calculationChoice, firstNumber, secondNumber);
+            var sumOfNumbers = Operations.CalculateSumOfNumbers(firstNumber, secondNumber);
 
-            WriteLine($"Sum: {sumOfNumbers}"); 
+            WriteLine($"{Environment.NewLine}Sum: {sumOfNumbers}"); 
```

Comment:
This will break two lines.

Reply:
In the console it's displaying on just a new line, without that there it displays on the same line as the input given by the user

Reply:
I didn't payed attention on the ReadKey... my bad.